Cargo.toml

```diff
@@ -56,7 +56,7 @@ rand = "0.7"
 regex = "1.5"
 rusqlite = "0.26"
 rust-hsluv = "0.1"
-rustyline = { version = "9.0", optional = true }
+rustyline = { version = "=9.0.0", optional = true }
```

Comment:
Rustyline is locked because of Rust 2021 dependency: https://github.com/kkawakam/rustyline/issues/581